.env.test

```diff
@@ -15,6 +15,7 @@ MAAT_API_OAUTH_URL='https://oauth.example.com'
 MAAT_API_CLIENT_ID='TestMaatApiClientID'
 MAAT_API_CLIENT_SECRET='TestMaatApiClientSecret'
 MAAT_API_API_URL='https://example.com'
+MAAT_API_STARTING_MAAT_ID=3
```

Comment:
This constant is set on a per environment basis and is used to prevent unsupported applications being retrieved.

Reply:
Rename constant and add explanation